### PR TITLE
Added open graph image for linkedin

### PR DESCRIPTION
**Pull request by devlife**
LinkedIn uses open graph image meta tags like this: `<meta name="image" property="og:image" content="{path} />` as can be seen in the [post inspector](https://www.linkedin.com/post-inspector/).

Happy to make any additional changes or follow a different approach.

### CODE REVIEW

File: MetaTags/Models/MetaTagsPart.cs [hunk 1]
```diff
@@ -1,16 +1,16 @@
-﻿using OrchardCore.ContentManagement;
```
Comment (devlife) [inline MetaTags/Models/MetaTagsPart.cs:1]: These changes were from line endings. I can undo if requested.

File: Etch.OrchardCore.SEO.csproj [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Version>0.7.0-rc2</Version>
+    <Version>0.7.1-rc2</Version>
```
Comment (peterkeating) [inline Etch.OrchardCore.SEO.csproj:7]: You'll also need to change the version number in the [`Manifest.cs`](https://github.com/EtchUK/Etch.OrchardCore.SEO/blob/master/Manifest.cs#L7) file.

File: MetaTags/Services/MetaTagsService.cs [hunk 3]
```diff
@@ -1,4 +1,5 @@
 ﻿using Etch.OrchardCore.Fields.Dictionary.Models;
+using Etch.OrchardCore.Fields.Eventbrite.Models.Dto;
```
Comment (peterkeating) [inline MetaTags/Services/MetaTagsService.cs]: Don't think you meant to add this reference?

File: Manifest.cs [hunk 4]
```diff
@@ -4,7 +4,7 @@
     Name = "SEO",
     Author = "Etch",
     Website = "https://etchuk.com",
-    Version = "0.7.0"
+    Version = "0.7.1-rc2"
```
Comment (peterkeating) [inline Manifest.cs]: Sorry the "rc2" postfix isn't needed.

**Review by peterkeating — CHANGES_REQUESTED** [review]

**Comment by devlife** [comment]
Removed `-rc2`

**Pull request merged**